tcf_website/management/commands/import_csv_review_drive.py

```diff
@@ -0,0 +1,227 @@
+                mnemonic__iexact=mnemonic
+            ).first()
+            if subdepartment is None:
+                print(
+                    "Subdepartment not found for",
+                    mnemonic,
+                    num,
+                    email,
+                    instructor_arr,
+                    "skipping...",
+                )
+                continue
+
+            course = Course.objects.filter(
+                subdepartment=subdepartment, number=num
+            ).first()
+            if course is None:
+                print(
+                    "Course not found for",
+                    mnemonic,
+                    num,
+                    email,
+                    instructor_arr,
+                    "skipping...",
+                )
+                continue
+
+            # Use only last name if list is size of 1
+            # Some professors only have last name in the database
+            if len(instructor_arr) > 1:
+                instructor = Instructor.objects.filter(
+                    first_name__iexact=instructor_arr[0],
+                    last_name__iexact=instructor_arr[1],
+                ).first()
+            else:
+                instructor = Instructor.objects.filter(
+                    last_name__iexact=instructor_arr[0]
+                ).first()
+
+            if instructor is None:
+                print(
+                    "Instructor not found for",
+                    mnemonic,
+                    num,
+                    email,
+                    instructor_arr,
+                    "skipping...",
+                )
+                continue
+
+            review_content = line[5].strip()
+
+            # Confirm the review is not already in the database
+            if Review.objects.filter(text__iexact=review_content).count() > 0:
+                print(
+                    "REVIEW ALREADY IN DATABASE FOR",
+                    mnemonic,
+                    num,
+                    email,
+                    instructor_arr,
+                    "skipping...",
+                )
+                continue
+
+            # CSV must be formatted EXACTLY in order
+            instructor_rating = int(line[6].strip())
+            enjoyability = int(line[7].strip())
+            recommendability = int(line[8].strip())
+            difficulty = int(line[9].strip())
+            amount_reading = int(line[10].strip())
+            amount_writing = int(line[11].strip())
+            amount_group = int(line[12].strip())
+            amount_homework = int(line[13].strip())
+            hours_per_week = (
+                    amount_reading + amount_writing + amount_group + amount_homework
+            )
+
+            created = datetime.strptime(line[0].strip(), "%m/%d/%Y %H:%M:%S")
+            modified = datetime.now()
+
+            if account == dummy_account:
+                review_email = email
+            else:
+                review_email = ""
```

Comment:
We setting this to an empty string because the user doesn't have an email, right - I think a comment clarifying this can't hurt.

Reply:
Also, it seems like we can avoid `review_email` all together - see comment on ~line 114